content/post/kubedb-v2023.08.18/index.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Announcing KubeDB v2023.08.18
+date: "2023-08-19"
+weight: 14
+authors:
+- Raihan Khan
+tags:
+- cloud-native
+- cruise-control
+- dashboard
+- database
+- day-2-operations
+- elasticsearch
+- kafka
+- kubedb
+- kubernetes
+- mariadb
+- mongodb
+- mysql
+- opensearch
+- percona
+- percona-xtradb
+- pgbouncer
+- postgresql
+- prometheus
+- proxysql
+- redis
+- security
+---
+
+We are pleased to announce the release of KubeDB v2023.08.18. This post lists all the major changes done in this release since the last release. The release includes -
+
+- **Use of the restricted pod security label** ⇒  Pod security policies has been removed in k8s 1.25. In place, It brings `Pod Security Standards` into the picture. We are using the restricted mode (https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted) to all of the namespaces where our operators will be installed.   Thus, we achieve some good security standards, like running as non-root-user, privilege escalation will not happen, some destructive kernel capabilities will be dropped etc.
+
+- **Uniform conditions across database opsRequests** ⇒ We utilize the status.Conditions section of the opsRequest CR for correctly maintaining the steps of an opsRequest like VersionUpdate, HorizontalScaling etc. These conditions have been made uniform in all of our supported databases now.
+
+- **Reduce get/patch api calls** ⇒ #TODO
```

Comment:
todo ?

Reply:
The description for Reducing Get/Patch calls changes is not added yet.